docs/cookbook/generating_captions.md

```diff
@@ -0,0 +1,49 @@
+# Generate Captions for an Image
+
+In this recipe, we go over how to use LLMs to generate a descriptive caption set of tags for an image with OpenAI’s `gpt-4o-mini`.
+
+??? info "Key Concepts"
+
+    - [Calls](ADD LINK)
+    - [Response Model](ADD LINK)
+
+!!! note "Background"
+
+    Caption generation evolved from manual human effort to machine learning techniques like Conditional Random Fields (CRFs) and Support Vector Machines (SVMs), which were time-consuming and resource-intensive. Large Language Models (LLMs) have revolutionized this field, enabling efficient multi-modal tasks through API calls and prompt engineering, dramatically improving caption generation speed and accuracy.
+
+!!! Warning
+
+    This recipe will only work for those which support images (OpenAI, Gemini, Anthropic) as of 8/13/2024. Be sure to check if your provider has multimodal support.
+
+With OpenAI’s multimodal capabilities, image inputs are treated just like text inputs, which means we can use it as context to ask questions or make requests. For the sake of reproducibility, we will get our image from a URL to save you the hassle of having to find and download an image. The image is [a public image from BBC Science of a wolf](https://www.sciencefocus.com/space/wolf-moon) in front of the moon (the link goes to the article, but use the URL of the actual image by right clicking the image and getting the image address).
```

Comment:
We should just include the direct URL instead of to the article, right? Why be indirect here?